src/InfoBoxes/Content/Factory.cpp

```diff
@@ -527,7 +527,7 @@ static constexpr MetaData meta_data[] = {
     IBFHelper<InfoBoxContentSpeedIndicated>::Create,
     e_Load_G, // G load
     e_Track_GPS, // Track
-    DEPRECATED,
+    STANDARD,
   },
```

Comment:
Are you able to tell me where this infobox appears such as setup -> screen -> infoboxes -> cruise -> infobox: 8 -> content? When I update to Android 15, I can't install the created apk file. Therefore, I can't find where.

Reply:
If you tap an existing infobox you can easily replace it with some other infobox, but the list that appears does not include all possible ones - the "deprecated" ones do not appear.  To assign a deprecated infobox to a spot on a screen you have to go through the settings / advanced / screens / etc process which is harder.  Since I see the IAS infobox as important (in an installation with no pitot data input to Tophat) I want to make it easier to reach, just a couple of taps from an existing infobox position in an existing screen.

Reply:
Could you download http://www.caztech.com/glider/tophat/android/XCSoar-debug.apk and try to install on your android phone/tablet? I have no testing environment. If you can, let me know whether infoboxes have IAS option.